src/web/modules/gateway_accounts/gateway_accounts.http.ts

```diff
@@ -215,6 +216,17 @@ const toggleBlockPrepaidCards = async function toggleBlockPrepaidCards(
   res.redirect(`/gateway_accounts/${id}`)
 }
 
+const toggleMotoPayments = async function toggleMotoPayments(
+  req: Request,
+  res: Response
+): Promise<void> {
+  const gatewayAccountId = req.params.id
+  await Connector.toggleMotoPayments(gatewayAccountId)
+
+  req.flash('info', 'Updated allow_moto')
```

Comment:
I think this should be a more human readable message, i.e. "MOTO payments enabled" or "MOTO payments disabled".

Reply:
This will be annoying to do as it's hard to pull out the "true" or "false" value from the form submission. I'll settle for 'Updated MOTO payments' which is more English-like if that's ok.